.github/workflows/elixir.yml

```diff
@@ -0,0 +1,29 @@
+on: [push, pull_request]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    name: Elixir
+    services:
+      db:
+        image: postgres:11
+        ports: ['5432:5432']
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    steps:
+      - uses: actions/checkout@v1.0.0
+      - uses: actions/setup-elixir@v1.0.0
+        with:
+          otp-version: 21.1
+          elixir-version: 1.7.4
+      - name: Install Dependencies
```

Comment:
Just a question: local.hex and local.rebar are already run during the setup?

Reply:
Yes, see https://github.com/actions/setup-elixir/blob/master/action.yml where `actions/setup-elixir` refers to. Installing `local.hex` and `local.rebar` defaults to true.

Reply:
Nice!